plugin/Field/Collection/FieldLoaderGeneric.php

```diff
@@ -80,7 +80,7 @@ public function load(): Generator
 			if (isset($fieldArray['label'])) {
 				unset($fieldArray['label']);
 			}
-			$listTypeUnSupported = ['user', 'datei', 'redhint', 'blackhint', 'dividingline'];
+			$listTypeUnSupported = ['user', 'date', 'redhint', 'blackhint', 'dividingline'];
```

Comment:
Did you check which one is correct? _datei_ is German for _file_, which is what is supposed to be removed in this project.

Reply:
This also applies to the other changes of the commit